### PR TITLE
Revert "Revert "SilentAlerts""

### PR DESCRIPTION
**Pull request by StephenBlackWasAlreadyTaken**
Reverts StephenBlackWasAlreadyTaken/xDrip-Experimental#48

Take 2!

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java [hunk 1]
```diff
@@ -265,15 +265,20 @@ public void notificationSetter(Context context) {
...
+        if(activeBgAlert != null ) {
+            AlertType alert = AlertType.get_alert(activeBgAlert.alert_uuid);
+            if(alert != null) {
+                int time = alert.minutes_between;
+                Calendar calendar = Calendar.getInstance();
+                AlarmManager alarm = (AlarmManager) getSystemService(ALARM_SERVICE);
+                if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.KITKAT) {
+                    alarm.setExact(AlarmManager.RTC_WAKEUP, calendar.getTimeInMillis() + time, PendingIntent.getService(this, 0, new Intent(this, Notifications.class), 0));
+                } else {
+                    alarm.set(AlarmManager.RTC_WAKEUP, calendar.getTimeInMillis() + time, PendingIntent.getService(this, 0, new Intent(this, Notifications.class), 0));
```
Comment (StephenBlackWasAlreadyTaken) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java]: note for self: set that `time`  badboy `(* 1000 * 60)`

Comment (tzachi-dar) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java]: Do you know what the issue is?
will you have a fix soon?
Should I work on this branch, to make my fix?

Thanks
Tzachi

Comment (StephenBlackWasAlreadyTaken) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java]: You can make yours off of master, I will clean this up tonight, I dont want to block you, sorry!

Comment (tzachi-dar) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java]: ok,
Thanks.

**Comment by swissalpine** [comment]
Hi Stephen,

thanks a lot: It fits perfectly to my needs!
I test it since one hour without any problems.

Andreas

**Pull request merged**